pkg/server/commands/check_command.go

```diff
@@ -23,6 +27,20 @@ import (
 const (
 	defaultResolveNodeLimit           = 25
 	defaultMaxConcurrentReadsForCheck = math.MaxUint32
+	defaultParentMethodName           = "check"
```

Comment:
I think this should be `""`. Otherwise, imagine you forget to call `WithParentMethodName` from Batch check or list objects, we will always think that these requests came from Check, when it reality it didn't. An empty string will allow us to know that we forgot to call `WithParentMethodName`